data/coalition jobs.txt

```diff
@@ -1105,7 +1105,7 @@ mission "Coalition: From Games 1"
 		"coalition jobs" ++
 		payment
 		payment 10000
-		dialog `The passengers thank you for bringing them to <planet> and pay you <payment>.`
+		dialog `The passengers thank you for bringing them to <planet> and pay you <payment>. From their overall manner, it looks like the Coalition Games were quite entertaining this year.`
```

Comment:
ah ok then the last part can be left in
so basically rephrase the start a little bit and you're good